pyproject.toml

```diff
@@ -9,13 +9,14 @@ docs = ["sphinx>=8.1", "sphinx-issues>=5", "furo"]
 lint = ["pre-commit"]
 typing = ["mypy", "packaging"]
 build = ["twine", "build"]
+dev = [{include-group = "coverage"}, {include-group = "test"}]
```

Comment:
Given that `test` includes coverage, I think just that one will suffice here.

Mostly out of curiosity: is it possible to configure `uv` to use the `test` group, or is it required to be `dev`?

Reply:
Sure, you can configure it. It just defaults to dev.

Oh, I didn’t notice. Sure.

Reply:
If you’d like the config instead, let me know. We are stuck with a little config until 3.8 is dropped due to the fact that it has a locking solver.

Reply:
Let's go with this for now. Learning how to adjust it might be a good "first real-world task with `uv`" for me.